src/app/sort/sort.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="row">
+  <div class="col">
+    <div ngbDropdown class="d-inline-block">
+      <button
+        type="button"
+        class="btn btn-outline-primary"
+        id="dropdownBasic1"
+        ngbDropdownToggle
+      >
+        {{ sortSelected }}
+      </button>
+      <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
```

Comment:
What is `dropdownBasic1`?

Investigate what `aria-labelledby` is used for and you will see that is not a proper naming!

Reply:
So, if I understood right,` aria-labelledby` is used for accessibility, right? So that's why it wouldn't be a good name, it doesn't help in accessibility at all. I'm changing it to sortingSelector, how does it sound?